src/database/queries.ts

```diff
@@ -32,7 +32,7 @@ const testColl = collection(db, 'test-col');
 const vendorCollection = collection(db, 'vendors');
 const voucherRangeCollection = collection(db, 'voucher-ranges');
```

Comment:
Can we rename 'voucher-range' to 'voucher-type' or something similar? It gets confusing especially now that we have the functionality for inputting a range of vouchers. If, for whatever reason the 'voucher-range' must stick, maybe we change everything related to range inputs to 'voucher-batch'.